src/storage/src/hummock/utils.rs

```diff
@@ -97,19 +97,6 @@ where
         .collect()
 }
 
-pub fn can_concat(ssts: &[&SstableInfo]) -> bool {
```

Comment:
I think we can keep this method and add the `debug_assert` to check whether the SST is able to concat when the `level.level_type == LevelType::Nonoverlapping`

Reply:
good idea